identity/aziot-identityd/src/identity.rs

```diff
@@ -670,13 +670,14 @@ impl IdentityManager {
                     Error::Internal(InternalError::CreateCertificate(Box::new(err)))
                 })?;
 
-                let _ = self
-                    .cert_client
-                    .create_cert(&identity_cert, &csr, None)
-                    .await
-                    .map_err(|err| {
-                        Error::Internal(InternalError::CreateCertificate(Box::new(err)))
-                    })?;
+                drop(
```

Comment:
Oh cool, a new `pedantic` clippy lint that I'm not a huge fan of.

See https://rust-lang.github.io/rust-clippy/master/#let_underscore_drop and the original PR https://github.com/rust-lang/rust-clippy/pull/6305

Personally, I feel like this lint is a bit _too_ aggressive. If I'm using `let _ =` on a value, I'm doing so because I don't care about it _at all_, and therefore don't care when/where it gets dropped. It seems like a truly pedantic generalization of the much more useful `let_underscore_lock` lint.

I'd personally vote to suppress this lint, but folks think we should keep it, so be it.

Reply:
Ugh, I didn't notice that rustfmt had expanded it to more lines. I'd initially typed `drop(self.client_client(` and `})?);` which was palatable.

I agree it's worth suppressing it.

Reply:
Done.